pkg/api/router.go

```diff
@@ -77,6 +77,21 @@ func GenerateRouter(apiConf *Config, metrics *Metrics, client *pgclient.Client,
 	router.Get("/debug/pprof/profile", pprof.Profile)
 	router.Get("/debug/pprof/symbol", pprof.Symbol)
 	router.Get("/debug/pprof/trace", pprof.Trace)
+	router.Get("/debug/pprof/heap", http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		pprof.Handler("heap").ServeHTTP(w, r)
+	}))
```

Comment:
```suggestion
	router.Get("/debug/pprof/heap", pprof.Handler("heap").ServeHTTP)
```
I think we can simplify it a bit

Reply:
Thank you for your review. this is now taken into account.

Reply:
Accepted, just rebase the branch on top of master so we can merge

Reply:
done, ready to be merged at your convenience.